clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntity.java

```diff
@@ -143,48 +144,48 @@ public void close() {
   public void validate(ServerSideConfiguration config) throws ClusteredTierManagerValidationException {
     try {
       invokeInternal(messageFactory.validateStoreManager(config), false);
-    } catch (Exception e) {
-      throw new ClusteredTierManagerValidationException("Error validating server clustered tier manager", e);
+    } catch (ClusteredClientException e) {
+      throw new ClusteredTierManagerValidationException("Error validating server clustered tier manager", e.getCause());
     }
   }
 
   public void configure(ServerSideConfiguration config) throws ClusteredTierManagerConfigurationException {
     try {
       invokeInternal(messageFactory.configureStoreManager(config), true);
-    } catch (Exception e) {
-      throw new ClusteredTierManagerConfigurationException("Error configuring clustered tier manager", e);
+    } catch (ClusteredClientException e) {
```

Comment:
This is very wrong - we end up wrapping a server exception in a client exception to unwrap it immediately and rewrap before throwing.